robot/robot.go

```diff
@@ -10,6 +10,7 @@ import (
 
 	"go.viam.com/rdk/config"
 	"go.viam.com/rdk/discovery"
+	modif "go.viam.com/rdk/module/modmanager/modmaninterface"
```

Comment:
can this cycle avoidance get moved to `rdk/module/`?

Reply:
Moved to rdk/module/modmaninterface, so it's one less layer deep. Still has to be its own package though, (in case you meant trying to merge it into module itself.)